easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,24 @@
+name = 'SWIG'
+version = '4.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e8a39cd6437e342cdcbd5af27a9bf11b62dc9efec9248065debcb8276fcbb925']
+
+dependencies = [
+    ('Python', '2.7.15'),
```

Comment:
I think we should really stick to python 3 as default when we have to pick.

Reply:
mhmmm, my $users of SWIG are on  `python-2.7.15` so that's my personal motivation behind.. Is this possible usecase for multi-deps ?

Reply:
Nope, we can't use multi-deps here (only applicable for pure binding packages or some softwares with python bindings), but SWIG actually bases it's target code generation on the python used during building (I think, at least there is nothing here where multideps makes sense as far as I can see in the final installation).

Can you add a python3 version as well? I think it's important to at least never have packages with just python2, to help alleviate any friction from our users to do the right thing and switch to python 3.

Reply:
>  to help alleviate any friction from our users to do the right thing

that's good argument. I'll do so.